tcms/rpc/tests/test_testplan.py

```diff
@@ -226,3 +228,123 @@ def test_ignores_existing_mappings(self):
                         case=case_id
                     ).count()
                 )
+
+
+class TestCreate(APITestCase):
+    def test_create_plan_with_empty_required_field(self):
+        product = ProductFactory()
+        version = VersionFactory(product=product)
+        plan_type = PlanTypeFactory(name='api testing')
+        with self.assertRaisesRegex(XmlRPCFault, 'name'):
```

Comment:
Error message must be more verbose.

Reply:
The error message may be translated in language other than English and may fail the job.

Reply:
That's true. For this reason API tests use @override_settings to avoid dealing with translations. Another option is to also to translate the error string, see test_testrun.py near the bottom.